bassist/bassist/parser/log_file/debs_stdout.py

```diff
@@ -21,13 +21,14 @@ def parse(self):
                 if line_number < 5: continue
 
                 parts = line.split()
-                print "parts: ", parts[2]
-                store_imports(" ".join(parts))
+                print "parts: ", parts
```

Comment:
Please remove `print` lines. OK to use them in debugging, but they shouldn't be in the committed code. If you *do* want debugging here in the final code, you should use `log` instead.